main.cc

```diff
@@ -356,9 +356,10 @@ std::string format_parsed_options(const std::vector<bpo::option>& opts) {
 }
 
 static constexpr char startup_msg[] = "Scylla version {} with build-id {} starting ...\n";
+static std::string build_id;
```

Comment:
There is now no guarantee that this is initialized before use. If calling get_build_id is too expensive (is it?) make this a static local variable in get_build_id:
```
string get_build_id() {
   static string ret = compute_build_id();
   return ret;
}

Reply:
ok